src/zonetree/zone.rs

```diff
@@ -13,9 +13,47 @@ use super::traits::WritableZone;
 use super::types::StoredName;
 use super::{parsed, ReadableZone, ZoneStore};
 
+//------------ ZoneKey -------------------------------------------------------
+
+/// A key that uniquely identifies a zone.
+///
+/// A zone is identified by the owner name of the apex and its class. Every
+/// record in a zone must be at or under the apex owner name and be of the
+/// same class.
+pub type ZoneKey = (StoredName, Class);
```

Comment:
`ZoneKey` is potentially confusing what with DNSSEC also using zone keys …

Reply:
Good point.

Reply:
Renamed to `ZoneTuple`.